graphql_client/__init__.py

```diff
@@ -50,7 +51,7 @@ def _conn_init(self, headers=None):
         self._conn.recv()
 
     def _start(self, payload):
-        _id = gen_id()
+        _id = uuid.uuid4()
```

Comment:
@TimGuiteDiamond have you tested this code? This throws a runtime exception, as the UUID object is not J SON-serializable. You need to fix this to:

```suggestion
        _id = uuid.uuid4().hex
```

Reply:
Sorry about this! I just made this change quickly and it looked like it returned a string, I'll make the change tomorrow.

I have some suggestions on how to do the multiple subscriptions but I'll leave them for a different PR/issue.

Thanks for responding and working on this 😄

Reply:
Have made this change now and tested it with my system.

Sorry for the extra commits, accidentally formatted it with black, but I reverted it so should just be the minimal changes now 👍